### ISSUE TITLE
mCRL2-GUI truncates filenames with dots

### ISSUE THREAD

**Issue by FerryT** [mCRL2org/mCRL2#1603]
Currently when using a file like <code>some.specification.dot.**mcrl2**</code> in mCRL2-GUI truncates it to, for instance, <code>some.**lts**</code>. Imho, it would be more convenient if it would truncate to <code>some.specification.dot.**lts**</code> instead. A similar case is true for mCRL2xi.

I have made the changes in [this branch](https://github.com/FerryT/mCRL2/tree/basename-fix).

**Comment by tneele** [comment]
I would support making these changes now. If you open a pull request, we can sneak it in before the upcoming release.

**Comment by tneele** [comment]
@Valo13 there are also a couple of occurrences of `baseName()` in `mcrl2ide/filesystem.cpp`. Perhaps you want to replace those with `completeBaseName()` as well.

**Comment by Valo13** [comment]
@tneele Yeah, this issue immediately reminded me of a [similar issue in mcrl2ide](https://github.com/mCRL2org/mCRL2/issues/1484#issuecomment-585819414) which I [fixed](https://github.com/mCRL2org/mCRL2/commit/571180d11f3bedac25397788ec1c9043369fe440), but I wasn't aware of the existence of `completeBaseName()`.
I don't know if `baseName()` causes actual issues elsewhere in mcrl2ide.
I'll maybe look at this somewhere next week; I want to spend the little energy I have left this week on my paper deadline of tomorrow.
